### PR TITLE
Chef manager retest

### PR DESCRIPTION
**Pull request by Skuxa**
Implement #77 after major rework.

### CODE REVIEW

**Review by dislocated-su — CHANGES_REQUESTED** [review]
You need to add javadoc comments and remove dangling commented out lines

File: tests/src/cs/eng1/tests/chef/ChefManagerTests.java [hunk 1]
```diff
@@ -1,98 +1,211 @@
...
+            );
+            if (chefIndex < 2) {
+                chefManager.hireChef(spawnPoints.get(chefIndex), stage);
+            }
+        }
+    }
+    // @Test
+    // public void loadFromSave() {
+    // SaveState state = SaveManager.getInstance().load("testsavefile.json");
+
```
Comment (dislocated-su) [inline tests/src/cs/eng1/tests/chef/ChefManagerTests.java:201]: Get rid of this

File: tests/src/cs/eng1/tests/customer/CustomerTests.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import org.junit.runner.RunWith;
+
+@RunWith(GdxTestRunner.class)
+public class CustomerTests {
+
+    // CustomerManager customerManager = new CustomerManager(0, null, null, 0);
+    Customer customer;
+
+    @Test
+    public void fulfillOrderTests() {}
```
Comment (dislocated-su) [inline tests/src/cs/eng1/tests/customer/CustomerTests.java:19]: Did we forget about this?

File: tests/src/cs/eng1/tests/stations/SubmitStationTests.java [hunk 3]
```diff
@@ -1,174 +1,188 @@
...
+    // * is
+    // * submitted, and that it moves onto the next order when the correct recipe is
+    // * submitted
+    // */
+    // @Test
+    // public void testDoStationAction() {
+    // customerManager = new CustomerManager(1, overlay, world, 0, 0);
+    // customerManager.init(
+    // textureManager,
+    // stage,
```
Comment (dislocated-su) [inline tests/src/cs/eng1/tests/stations/SubmitStationTests.java:153]: bruh

**Review by dislocated-su — CHANGES_REQUESTED** [review]
Please remove commented out tests

File: tests/src/cs/eng1/tests/customer/CustomerTests.java (hunk 2 above)
Comment (Skuxa) [reply tests/src/cs/eng1/tests/customer/CustomerTests.java]: yes

File: tests/src/cs/eng1/tests/chef/ChefManagerTests.java (hunk 1 above)
Comment (Skuxa) [reply tests/src/cs/eng1/tests/chef/ChefManagerTests.java]: yes

File: tests/src/cs/eng1/tests/stations/SubmitStationTests.java (hunk 3 above)
Comment (Skuxa) [reply tests/src/cs/eng1/tests/stations/SubmitStationTests.java]: yes.

**Review by dislocated-su — APPROVED** [review]

**Review by dislocated-su — APPROVED** [review]
This good

**Pull request merged**